src/nvim/testdir/test_edit.vim

```diff
@@ -14,6 +14,7 @@ func! Test_edit_00b()
   call setline(1, ['abc '])
   inoreabbr <buffer> h here some more
   call cursor(1, 4)
+  set startofline
```

Comment:
wouldn't this make more sense in `src/nvim/testdir/setup.vim` ?

Reply:
agreed & done.